core/internal/features_test.go

```diff
@@ -992,112 +986,6 @@ func TestIntegration_FluxMonitor_NewRound(t *testing.T) {
 	eth.EventuallyAllCalled(t)
 }
 
-func TestIntegration_RandomnessRequest(t *testing.T) {
```

Comment:
Github fail here! Because you moved this test function it's difficult to see the changes made in the new one. 
One way to fix this is to have all the changes made in this file in a commit, then another commit where you move the test to other file.

Reply:
oh that's a good suggestion :+1: I'll try to remember that for next time